source/change-streams/tests/unified/change-streams.yml

```
@@ -177,3 +217,585 @@ tests:
                 comment: "comment"
               commandName: getMore
               databaseName: *database0
+
+  - description: $changeStream must be the first stage in a change stream pipeline sent to the server
+    runOnRequirements:
+      - minServerVersion: 3.6.0
```

```suggestion
      - minServerVersion: "3.6.0"
```
Here and elsewhere in file.

---

Done.